launcher/LaunchController.cpp

```diff
@@ -144,6 +144,12 @@ void LaunchController::login()
     bool tryagain = true;
     unsigned int tries = 0;
 
+    if (m_accountToUse->accountType() != AccountType::Offline && m_accountToUse->accountState() == AccountState::Offline) {
+        // Force account refresh on the account used to launch the instance updating the AccountState
+        //  only on first try and if it is not meant to be offline
+        auto accounts = APPLICATION->accounts();
+        accounts->requestRefresh(m_accountToUse->internalId());
+    }
```

Comment:
shouldn't this be removed as it's in #1771

Reply:
This is a blocked by the mentioned PR, please review it after that one was merged. Initially this was supposed to be one single PR but due to scrumplex request I removed the auto reauth part from that one( hope that one is merged faster that way). Because I thought that the auto reauth thing is still useful I open an additional PR with the revert of that last commit. So once that one is merged we can discuss here if we want that or not.

Reply:
Oh. got it. I don't believe this has any reason to be based on the previous pr so I assumed it wasn't.